lib/AssemblyUtil/AssemblyUtilClient.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
```

Comment:
Right now dynamic services cannot call other local functions, so anything that does will not work.  So unfortunately we cannot use AssemblyUtil and GenomeFileUtil.  Fortunately, I don't think we need these for returning the Set?